src/plugins/data/common/query/timefilter/get_time.ts

```diff
@@ -64,17 +76,28 @@ function createTimeRangeFilter(
     return;
   }
 
-  const bounds = calculateBounds(timeRange, { forceNow });
-  if (!bounds) {
-    return;
+  let rangeFilterParams: RangeFilterParams = {
+    format: 'strict_date_optional_time',
+  };
+
+  if (coerceRelativeTimeToAbsoluteTime) {
+    const bounds = calculateBounds(timeRange, { forceNow });
+    if (!bounds) {
+      return;
+    }
+    rangeFilterParams.gte = bounds.min?.toISOString();
+    rangeFilterParams.lte = bounds.max?.toISOString();
+  } else {
+    rangeFilterParams.gte = isRelativeTime(timeRange.from)
+      ? timeRange.from
+      : calculateLowerBound(timeRange.from, forceNow)?.toISOString();
```

Comment:
is this needed if we are working with absolute time ? or could we always just take timeRange.from ?

Reply:
My assumption is yes. Some tests give values like `Jan 1, 2019 @ 00:00:00.000` for absolute time which ES does not understand and so `moment` was converting these to ISO strings.